rosjava/src/main/java/org/ros/node/topic/TransportHints.java

```diff
@@ -0,0 +1,43 @@
+package org.ros.node.topic;
+
+import java.util.Map;
+
+import org.ros.internal.transport.ConnectionHeaderFields;
+import org.ros.node.ConnectedNode;
+
+import com.google.common.collect.Maps;
+
+
+/**
+ * Provides a way of specifying network transport hints to
+ * {@link ConnectedNode#newSubscriber(String, String)} and
+ * {@link ConnectedNode#newSubscriber(org.ros.namespace.GraphName, String)}.
+ *
+ * @author stefan.glaser@hs-offenburg.de (Stefan Glaser)
+ */
+public class TransportHints {
+
+  Map<String, String> options;
+
+ public TransportHints() {
```

Comment:
nitpick: please indent functions and code with two spaces (same applies for next line).